src/Mixer/NullMixer.cpp

```diff
@@ -0,0 +1,50 @@
+#include "NAS2D/Mixer/NullMixer.h"
+
+namespace NAS2D
+{
+
+	void NullMixer::playSound(Sound& /*sound*/)
+	{}
+
+	void NullMixer::stopSound()
+	{}
+
+	void NullMixer::pauseSound()
+	{}
+
+	void NullMixer::resumeSound()
+	{}
+
+	void NullMixer::stopMusic()
+	{}
+
+	void NullMixer::pauseMusic()
+	{}
+
+	void NullMixer::resumeMusic()
+	{}
+
+	void NullMixer::fadeInMusic(Music& /*music*/, int /*loops*/ /*= Mixer::CONTINUOUS*/, int /*time*/ /*= Mixer::DEFAULT_FADE_TIME*/)
+	{}
+
+	void NullMixer::fadeOutMusic(int /*time*/ /*= Mixer::DEFAULT_FADE_TIME*/)
+	{}
+
+	bool NullMixer::musicPlaying() const
+	{
+		return false;
+	}
+
+	void NullMixer::mute()
+	{}
+
+	void NullMixer::unmute()
+	{}
+
+	void NullMixer::soundVolume(int /*level*/)
+	{}
+
+	void NullMixer::musicVolume(int /*level*/)
+	{}
+
+} // namespace NAS2D
```

Comment:
No EOL (End Of Line) at EOF (End Of File).

Pretty minor. Mostly just the GitHub visual warning, and if you `cat` the file at the Linux terminal, your next prompt may appear at the end of the last line of the file, rather than on a newline by itself.

There's also some history to it:
https://stackoverflow.com/questions/72271/no-newline-at-end-of-file-compiler-warning
https://stackoverflow.com/questions/31574837/c-standards-newline-ending-of-source-files

Older C++ standards used to require that newline. In the case of C, it might still be required.